src/objects/zcl_abapgit_object_clas.clas.abap

```diff
@@ -245,6 +308,27 @@ CLASS ZCL_ABAPGIT_OBJECT_CLAS IMPLEMENTATION.
     io_xml->add( iv_name = 'TPOOL'
                  ig_data = add_tpool( lt_tpool ) ).
 
+    IF io_xml->i18n_params( )-serialize_master_lang_only IS INITIAL.
+      CLEAR: lt_i18n_tpool[].
+      LOOP AT mt_langu_additional INTO lv_langu.
+        CLEAR: ls_i18n_tpool.
+
+        lt_tpool = mi_object_oriented_object_fct->read_text_pool(
+              iv_class_name = ls_clskey-clsname
+              iv_language   = lv_langu ).
+
+        ls_i18n_tpool-language = lv_langu.
+        ls_i18n_tpool-textpool = add_tpool( lt_tpool ).
+        INSERT ls_i18n_tpool INTO TABLE lt_i18n_tpool.
+
+      ENDLOOP.
+
+      IF lines( lt_i18n_tpool ) > 0.
+        io_xml->add( iv_name = 'I18N_TPOOL'
+                     ig_data = lt_i18n_tpool ).
+      ENDIF.
+    ENDIF.
```

Comment:
perhaps move this part to a separate method?

Reply:
Maybe do same that DESERIALIZE and separate different object is each method
`  METHOD zif_abapgit_object~serialize_xml.
    serialize_abap(  ).

    serialize_tpool( ).

    serialize_sotr(  ).

    serialize_docu( ).
  ENDMETHOD.`

Reply:
Encapsulated in different methods